tests/integration/ceph_mgr_test.go

```diff
@@ -122,19 +115,68 @@ func (s *CephMgrSuite) execute(command []string) (error, string) {
 	return s.installer.Execute("ceph", orchCommand, s.namespace)
 }
 
+func (s *CephMgrSuite) enableOrchestratorModule() {
+	logger.Info("Enabling Rook orchestrator module: <ceph mgr module enable rook --force>")
+	err, output := s.installer.Execute("ceph", []string{"mgr", "module", "enable", "rook", "--force"}, s.namespace)
+	logger.Infof("output: %s", output)
+	if err != nil {
+		logger.Infof("Not possible to enable rook orchestrator module: %q", err)
```

Comment:
shouldn't we return after the failure? Or do we intentionally continue with the below command?
```suggestion
		logger.Infof("failed to enable rook orchestrator module: %q", err)
		return
```